exec/java-exec/src/test/java/org/apache/drill/exec/store/pcapng/TestPcapngRecordReader.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.pcapng;
+
+import org.apache.drill.PlanTestBase;
+import org.apache.drill.common.exceptions.UserRemoteException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.nio.file.Paths;
+
+public class TestPcapngRecordReader extends PlanTestBase {
+  @BeforeClass
+  public static void setupTestFiles() {
+    dirTestWatcher.copyResourceToRoot(Paths.get("store", "pcapng"));
+  }
+
+  @Test
+  public void testStarQuery() throws Exception {
+    Assert.assertEquals(123, testSql("select * from dfs.`store/pcapng/sniff.pcapng`"));
+    Assert.assertEquals(1, testSql("select * from dfs.`store/pcapng/example.pcapng`"));
+  }
+
+  @Test
+  public void testCountQuery() throws Exception {
+    Assert.assertEquals(1, testSql("select count(*) from dfs.`store/pcapng/sniff.pcapng`"));
+    Assert.assertEquals(1, testSql("select count(*) from dfs.`store/pcapng/example.pcapng`"));
```

Comment:
I will rewrite this test because this it doesn't check what result this query will return. And I have a question. How can I write a test to check how many vectors are allocated for a query?